common/supplemental/windowsZones.xml

```diff
@@ -569,13 +570,12 @@ For terms of use, see http://www.unicode.org/copyright.html
 			<mapZone other="Nepal Standard Time" territory="001" type="Asia/Katmandu"/>
 			<mapZone other="Nepal Standard Time" territory="NP" type="Asia/Katmandu"/>
 
-			<!-- (UTC+06:00) Astana -->
-			<mapZone other="Central Asia Standard Time" territory="001" type="Asia/Almaty"/>
+			<!-- (UTC+06:00) Astana --> <!-- Microsoft probably keeps Central Asia Standard Time, but change Astana to something else. -->
+			<mapZone other="Central Asia Standard Time" territory="001" type="Asia/Bishkek"/>
```

Comment:
Almaty will be no longer UTC+6. Microsoft uses display text `(UTC+06:00) Astana`. Astana is the capital of Kazakhstan, but TZ database uses most populous city in the region (in this case, Almaty). Because Almaty moved to UTC+5, it cannot be the default mapping of this windows zone. I'm sure Microsoft preserves this zone, but changing the display text.

I usually pick the most populous region as 001 mapping. If we use this rule, Urumqi is the one among these zones. However, official zone for Urmuqi is actually China Standard Time (UTC+8). UTC+6 is used as "defacto" offset there. For this reason, Urmuqi might not be the zone used for 001 mapping. I picked Bishkek here. This is the second populous region among these regions.